posts/inside-rust/2023-02-22-governance-reform-rfc.md

```diff
@@ -9,6 +9,6 @@ team: leadership chat <https://www.rust-lang.org/governance>
 
 As part of [ongoing work on governance](https://blog.rust-lang.org/inside-rust/2022/10/06/governance-update.html), the "leadership chat" established a smaller "governance reform" working group to create an RFC to establish new project wide governance. This RFC is now live and can found on the RFCs repo: <https://github.com/rust-lang/rfcs/pull/3392>
 
-We have set up a [Zulip stream](https://rust-lang.zulipchat.com/#narrow/stream/369838-rfc-leadership-council-feedback) to provide an additional dedicated place for feedback on the RFC. In addition, team leads will be asking their respective teams for feedback directly. However, anyone is welcome to participate and provide feedback directly in the Zulip stream or RFC repository if they prefer.
+We have set up a [Zulip stream](https://rust-lang.zulipchat.com/#narrow/stream/369838-rfc-leadership-council-feedback) to provide an additional dedicated place for feedback on the RFC. In addition, team leads will be asking their respective teams for feedback directly. However, anyone is welcome to participate and provide feedback directly in the Zulip stream or the RFC PR if they prefer.
```

Comment:
I thought the point was that we didn't want people commenting on the PR directly, because it gets too chaotic? The PR says

> For discussion on this PR, please use [this dedicated Zulip stream](https://rust-lang.zulipchat.com/#narrow/stream/369838-rfc-leadership-council-feedback).

Reply:
@jyn514 We'd prefer comments to generally go to the Zulip stream, but (for instance) suggestions and fixes, such as your helpful fixes for link URLs, make sense to post to the PR directly. We can update this text further if that changes.